bigtable/tests/run_integration_tests.sh

```diff
@@ -63,3 +63,7 @@ echo
 echo "Running Table::Apply() integration test."
 # The project and instance do not matter for the Cloud Bigtable emulator.
 ./integration_test emulated emulated test-table
+
+echo
+echo "Running TableAdmin integration test."
+./admin_test emulated admin-tests
```

Comment:
File names are confusing. They are both integration tests. Can we change them to data_integration_test and admin_integration_test or at least something consistent?

Reply:
Done.